src/client/src/rt-theme/index.ts

```diff
@@ -2,9 +2,9 @@ export { flush, hydrate, cx, merge, getRegisteredStyles, injectGlobal, keyframes
 
 import './globals'
 
-export { colors } from './colors'
+export { colors, CorePalette, CorePaletteMap, ColorPalette, ColorPaletteMaps } from './colors'
 export { themes, Theme } from './themes'
-export { resolvesColor } from './tools'
+export { resolvesColor, Selector } from './tools'
 export { ThemeProvider } from './ThemeProvider'
 export { ThemeState, ThemeStateValue } from './ThemeState'
-export { default as styled } from './styled'
+export { default as styled, Styled, StyledProps } from './styled'
```

Comment:
You shouldn't need `Styled` or `StyledProps`.

Reply:
There are two points to address:
1. Some APIs entirely fail when employing the `styled.div<props>` approach. I would encourage anyone finding a solution.
2. This approach is consistent with the typing of any other non-class component
```ts
const SomeComponent: React.SFC<SomeComponentProps> = () => 
```

Reply:
1. That makes sense.
2. TypeScript is able to infer from the styled function. In the example you provide, TypeScript cannot infer that this function is specifically `React.SFC`, hence the explicit type.